Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1050,6 +1050,10 @@
 #define ENABLE_WK_WEB_EXTENSIONS_SIDEBAR 0 && ENABLE_WK_WEB_EXTENSIONS
 #endif
 
+#if !defined(ENABLE_WK_WEB_EXTENSIONS_BOOKMARK)
```

Comment:
`ENABLE_WK_WEB_EXTENSIONS_BOOKMARKS` sounds better and the singular for.

```suggestion
#if !defined(ENABLE_WK_WEB_EXTENSIONS_BOOKMARKS)
```